_aggregations/metric/extended-stats.md

```diff
@@ -73,4 +113,145 @@ GET opensearch_dashboards_sample_data_ecommerce/_search
     }
   }
 }
-```
\ No newline at end of file
+```
+{% include copy-curl.html %}
+
+This changes the standard deviation bounds:
+
+```json
+{
+...
+  "aggregations": {
+...
+      "std_deviation_bounds": {
+        "upper": 233.44837084770438,
+        "lower": -83.33751356160813,
+        "upper_population": 233.44837084770438,
+        "lower_population": -83.33751356160813,
+        "upper_sampling": 233.46531398752978,
+        "lower_sampling": -83.35445670143353
+      }
+    }
+  }
+}
+```
+
+## Missing values
+
+You can assign a value to missing instances of the aggregated field. See [Missing aggregations]({{site.url}}{{site.baseurl}}/aggregations/bucket/missing/).
+
+Prepare an example index by ingesting the following documents:
+
+```json
+POST _bulk
+{ "create": { "_index": "students", "_id": "1" } }
+{ "name": "John Doe", "gpa": 3.89, "grad_year": 2022}
+{ "create": { "_index": "students", "_id": "2" } }
+{ "name": "Jonathan Powers", "grad_year": 2025 }
+{ "create": { "_index": "students", "_id": "3" } }
+{ "name": "Jane Doe", "gpa": 3.52, "grad_year": 2024 }
+```
+{% include copy-curl.html %}
+
+### Example: Replacing a missing value
+
+Compute `extended_stats`, replacing the missing GPA field with `0`:
+
+```json
+GET students/_search
+{
+  "size": 0,
+  "aggs": {
+    "extended_stats_gpa": {
+      "extended_stats": {
+        "field": "gpa",
+        "missing": 0
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+The response is as follows. Compare to the next example, where missing values are ignored:
+
```

Comment:
Line 176: Same comment re: context. Something like "In the response, the missing value is replaced with...."